app/src/main/java/io/intrepid/contest/screens/entrysubmission/entryimage/EntryImageActivity.java

```diff
@@ -186,8 +214,24 @@ public void showContestStatusScreen() {
         startActivity(ContestStatusActivity.makeIntent(this));
     }
 
+    @RequiresApi(api = Build.VERSION_CODES.M)
+    @Override
+    public void checkStoragePermissions() {
```

Comment:
Good point. Crashed pre-m. No method for checkPermissions. I'll need to return true if below M. Thanks!